### PR TITLE
added time change and actor list

### PR DESCRIPTION
**Pull request by Nickpopadopolous**

### CODE REVIEW

**Comment by dzimmerman902** [comment]
Looks good so far

File: assets/javascript/onConnectAPI.js [hunk 1]
```diff
@@ -43,10 +43,6 @@ $(document).ready(function () {
 
 					var theatreName = movie.showtimes[0].theatre.name;
 					$('#theatreName').append(theatreName);
-
-					var actors = movie.topCast;
```
Comment (dzimmerman902) [inline assets/javascript/onConnectAPI.js]: I see where got the APIs mixed up lol

**Comment by AdamJahur** [comment]
so we only getting the 5 main movie stars and the time format looks good and im assuming its working fine

**Comment by Nickpopadopolous** [comment]
I mean it is easy to add more movie stars in it. At first it takes all the cast and the list was like 100 people lol

**Pull request merged**